### DISCUSSION TITLE
code-hike v1 with nextra v3

### DISCUSSION THREAD

**Discussion by ZYJLiu** [code-hike/v1#9] [Q&A]
Is it possible to use codehike v1 with nextra? Currently having trouble figuring out how to set up the next.config.mjs file.
Trying to use the new scrollycode with token transitions in a nextra project.

**Comment by pomber** [comment]
There's still a blocking issue with Nextra v3. You can follow https://github.com/code-hike/examples/pull/6 for updates.
